api4/channel_test.go

```diff
@@ -1849,6 +1849,8 @@ func TestGetChannelStats(t *testing.T) {
 		t.Fatal("couldnt't get extra info")
 	} else if stats.MemberCount != 1 {
 		t.Fatal("got incorrect member count")
+	} else if stats.PinnedPostCount != 0 {
+		t.Fatal("got incorrect pinned post count")
```

Comment:
Thanks for adding but could you add another case where there is at least a number of pinned post/s?

Reply:
@saturninoabril I added the test!

Reply:
Thanks!